tests/test_training.py

```diff
@@ -96,3 +109,43 @@ def test_ipu_cli_training(self, load_type):
 
                 assert _IPU_AVAILABLE is True
                 self.call_cli_with_overrides("ipu", "16-true", load_type)
+
+    def test_ipu_mlp(self):
+
+        import torch
+        from torch import nn
+        import poptorch
+
```

Comment:
```suggestion
       with patch("poptorch.ipuHardwareIsAvailable", return_value=True):
            import torch
            from torch import nn
            import poptorch
            assert poptorch.ipuHardwareIsAvailable()
            
            ...
            rest of code here inside context manager

```

Reply:
Should I also override the `_IPU_AVAILABLE` value from Lightning here?